app/src/layouts/DashboardLayout/AppNotificationBanner/types.ts

```diff
@@ -0,0 +1,4 @@
+export enum BANNER_ACTIONS {
```

Comment:
do these need to be exported? will they be consumed elsewhere? if not, can we also make these part of the original file near the `Banner` interface